### ISSUE TITLE
508 Accessibility compliance

### ISSUE THREAD

**Issue by gerardlopez2001** [QMDevTeam/QMAndroid#14]
For all user facing application features such as screens and menus, Section 508 Accessibility features should be enabled.

For more information, please see

http://developer.android.com/guide/topics/ui/accessibility/index.html

Related to QMDevTeam/QMGeneral#15

Related to #6

**Comment by gerardlopez2001** [comment]
Moved to Trello: https://trello.com/c/gaAbisEi
